nautobot/extras/templates/extras/objectchange.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'generic/object_retrieve.html' %}
-{% load helpers %}
+{% load helpers static %}
```

Comment:
TIL. I honestly didn't know this form was doable - all of our templates that I'm aware of have separate `load` statements for each set of template tags.